lib/js/components/RichList/RichListItem/RichListItem.vue

```diff
@@ -391,8 +473,38 @@ export default {
 			type: String,
 			default: null,
 		},
+		backgroundColor: {
+			type: String as PropType<RichListItemBackgroundColor>,
+			default: RICH_LIST_ITEM_BACKGROUND_COLOR.NEUTRAL,
+			validator(backgroundColor) {
+				return Object.values(RICH_LIST_ITEM_BACKGROUND_COLOR).includes(backgroundColor);
+			},
+		},
+		elevation: {
+			type: String as PropType<RichListItemElevation>,
+			default: null,
+			validator(evolution) {
+				return Object.values(RICH_LIST_ITEM_ELEVATION).includes(evolution);
+			},
+		},
+		hasDraggableHandler: {
```

Comment:
What is the purpose of this prop?

Reply:
Decision from last meeting, row can be moved by the displayed icon or by the entire element.